src/analysis/classic/hunter/survival/modules/spells/SerpentSting.tsx

```diff
@@ -0,0 +1,74 @@
+import { t } from '@lingui/macro';
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS/classic/hunter';
+import { SpellLink } from 'interface';
+import { SpellIcon } from 'interface';
```

Comment:
You can combine lines 4 & 5 into 1 line:
`import { SpellLink, SpellIcon } from 'interface';`

Reply:
Done